Q/QGRAF/build_tarballs.jl

```diff
@@ -0,0 +1,63 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "QGRAF"
+version = v"3.6.5"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("http://anonymous:anonymous@qgraf.tecnico.ulisboa.pt/v3.6/qgraf-$(version).tgz", "648cc82bf3327a4c36d36847d3921fcfa5d7f73c1646241760db030abdc86c29")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+
+cat sha256sums | while read line; do
```

Comment:
What is this doing?

Reply:
Check the `sha256sum`.
The `.tgz` includes a file archived all information about the files.

Reply:
Do you think this part is useless?

Reply:
Yeah, I mean, we are already check summing the entire tarball.

Reply:
Emm, that sounds enough. I agree it.